### DISCUSSION TITLE
ubuntu多用户的服务器上认证超时

### DISCUSSION THREAD

**Discussion by Raywoo1** [houtianze/bypy#642] [Q&A]
请问我一直认证超时咋办？是在一台乌班图的服务器上，服务器上有好几个用户。如果别的用户也用bypy，会不会导致冲突，最终导致我认证失败呢？我第一次用的时候就在前几天，是可以认证成功的。现在就认证超时了

**Comment by houtianze** [comment]
升级最新版试试？最新版已经不需要服务器认证了。不同用户的话，令牌文件存放位置不一样的，应该不会冲突。

**Reply by Raywoo1** [reply]
已经是最新版了，我bypy info -ddd了一下，输入验证码后是这样
![image](https://user-images.githubusercontent.com/28223203/227078283-4101e017-dbba-4dc5-a6b0-3ed17c64a8d4.png)

**Comment by houtianze** [comment]
看了一下，这里返回都是200，应该没有什么问题。问题出在其他地方？

**Reply by Raywoo1** [reply]
我也感觉奇怪，下面是完整的一个截图
![image](https://user-images.githubusercontent.com/28223203/227080777-b81a400e-f112-4822-ba6e-cfa8d4e95aef.png)

**Reply by Raywoo1** [reply]
> 看了一下，这里返回都是200，应该没有什么问题。问题出在其他地方？
> […](#)
> On Thu, Mar 23, 2023 at 9:49 AM Raywoo ***@***.***> wrote: 已经是最新版了，我bypy info -ddd了一下，输入验证码后是这样 [image: image] <https://user-images.githubusercontent.com/28223203/227078283-4101e017-dbba-4dc5-a6b0-3ed17c64a8d4.png> — Reply to this email directly, view it on GitHub <[#642 (reply in thread)](https://github.com/houtianze/bypy/discussions/642#discussioncomment-5399624)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAYFZSOF52JFJAISJXYX5DW5OT4PANCNFSM6AAAAAAWERZ22A> . You are receiving this because you commented.Message ID: ***@***.***>

然后超时的报错是这个
![image](https://user-images.githubusercontent.com/28223203/227081211-27276623-0841-4f61-883b-4054d6e17f6c.png)

**Comment by houtianze** [comment]
这个应该与授权无关，读取超时重试而已。

**Comment by Anutlet** [comment]
第一次认证失败，用的也是最新版本，系统环境是多用户ubuntu，怎么排查问题呢
![image](https://github.com/houtianze/bypy/assets/15341892/f039779d-c452-436a-a731-0d9c94d222b2)

**Reply by Anutlet** [reply]
@houtianze

**Comment by houtianze** [comment]
`bypy -d -v`试一下
